epochStart/shardchain/trigger.go

```diff
@@ -42,11 +43,13 @@ type ArgsShardEpochStartTrigger struct {
 type trigger struct {
 	currentRoundIndex           int64
 	epochStartRound             uint64
+	epochStartNonce             uint64
```

Comment:
This could be removed as you have this info in epochStartMeta

Reply:
done